### PR TITLE
refactor(docs): switch to unocss

### PR DESCRIPTION
**Pull request by sxzz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by YunYouJun** [comment]
Use `presetIcons` too?

> https://github.com/antfu/vitesse/blob/main/unocss.config.ts

**Comment by sxzz** [comment]
@YunYouJun Done.

**Review by YunYouJun — APPROVED** [review]

**Comment by YunYouJun** [comment]
It seems to be some conflict that needs to be resolved.

File: docs/unocss.config.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    colors: {
+      primary: {
+        DEFAULT: '#2563eb',
+        deep: '#1d4ed8',
+      },
+    },
+    breakpoints: {
+      xs: '320px',
+      sm: '640px',
+    },
```
Comment (JeremyWuuuuu) [inline docs/unocss.config.ts:17]: **Question:** does this match the breakpoint of our application?

CleanShot 2022-03-21 at 21 50 41@2x

**Review by jw-foss — COMMENTED** [review]
@sxzz I left a comment for you, the rest of your code LGTM. 🏓

File: docs/unocss.config.ts (hunk 1 above)
Comment (sxzz) [reply docs/unocss.config.ts]: Updated.

**Pull request merged**